doc/user/content/release-notes.md

```diff
@@ -252,6 +252,8 @@ Wrap your release notes at the 80 character mark.
   - Handle parsing [`timestamp`](/sql/types/timestamp) and [`timestamptz`](/sql/types/timestamptz)
     from additional compact formats like `700203` {{% gh 4889 %}}.
 
+  - Support conversion of [`timestamp`](/sql/types/timestamp) and [`timestamptz`](/sql/types/timestamptz) to other time zones with [`AT TIME ZONE`](/sql/functions/#date-and-time-func) and [`timezone`](/sql/functions/#date-and-time-func) functions.
```

Comment:
"conversion of" is either passive voice or feels like it; using verbs makes writing feel much less plodding and monolithic, e.g. "Supporting converting `timestamp`...."

Reply:
"Conversion of timestamp" is the object phrase, not the verb; the verb is "Support", which is in the active voice. I think "Support converting" or "Handle converting" sound awkward, so I'm leaving as-is.

Reply:
Ah, I typoed my suggestion, which should have been "Support converting `timestamp`...". You'll notice this is the format of most other release notes. I believe you've used the argument that consistent formatting is of paramount importance, so not sure why that doesn't apply here?